ui/storybook/src/utils/requests/queryRangeRequests.ts

```diff
@@ -42,9 +42,6 @@ export function mockQueryRangeRequests({ queries }: MockQueryRangeConfig): Reque
       const requestQuery = typeof requestPostData === 'object' ? requestPostData['query'] : undefined;
       const mockQuery = queries.find((mockQueryConfig) => mockQueryConfig.query === requestQuery);
 
-      console.log(requestQuery);
-      console.log(queries);
```

Comment:
I also accidentally left in a few console logs.

Reply:
I wonder if we should have an eslint error for this? Maybe something for a follow-up.

Reply:
yeah I think that would be cool. I usually forget them too